packages/jest/lib/ExpressionEvaluator.ts

```diff
@@ -0,0 +1,61 @@
+import { BindingsFactory } from '@comunica/bindings-factory';
```

Comment:
Can we name this file `expressions.ts`? Otherwise, it feels like this class exposes the `ExpressionEvaluator` class.